### PR TITLE
Include schema name on each change

### PR DESCRIPTION
**Pull request by ruimarinho**
This PR introduces the schema name on each change so that there is no need to store state between changes.

Not sure if "schema name" or "table name" is more appropriate for this PR.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -122,9 +130,8 @@ unsafe fn append_tuple_buf_as_json(data: *mut pg::ReorderBufferTupleBuf,
         let heap_tuple = &mut (*data).tuple;
         let datum = pg::heap_copy_tuple_as_datum(heap_tuple, desc);
         let empty_oid: pg::Oid = 0;
-        let json = pg::DirectFunctionCall1Coll(Some(row_to_json),
-                                               empty_oid,
-                                               datum);
+        let json =
+            pg::DirectFunctionCall1Coll(Some(row_to_json), empty_oid, datum);
```
Comment (ruimarinho) [inline src/lib.rs:134]: `rustfmt` was complaining about this, but I can add on a different PR.

Comment (solidsnack) [inline src/lib.rs:134]: Totally fine. I remember having seen that one. Thank you for fixing it.

**Comment by solidsnack** [comment]
We should flip the order of `table` and `schema` in the schema lines, so that the unique, indicative token is first in all lines.

**Comment by posix4e** [comment]
+1

**Comment by posix4e** [comment]
Err +1 on solid's comment

**Comment by ruimarinho** [comment]
I will do that then. Not sure why the tests are failing on travis - they pass locally. I'll take a look.

**Comment by solidsnack** [comment]
@ruimarinho It fails because some tests are missing relative to master. If you rebase and include results from the new tests it should fix the issue.

**Comment by posix4e** [comment]
Please rebate this

**Comment by ruimarinho** [comment]
Rebased and also inverted order of schema and table fields.

**Pull request merged**